src/modules/hardware-wallet/src/LedgerApp.ts

```diff
@@ -0,0 +1,221 @@
+// https://github.com/radixdlt/radixdlt-ledger-app/blob/improve/change_cosmos_to_radix/docs/APDUSPEC.md
+
+import { ReturnCode, Instruction, CLA, AppState } from './types'
+import { sendApduMsg } from './HardwareWallet'
+import { cborByteOffsets } from './atomByteOffsetMetadata'
+import { Subject, Observable } from 'rxjs'
+import { RadixAtom, RadixTransferrableTokensParticle, RadixSpin, RadixECSignature, RadixBytes, RadixAddress } from 'radixdlt'
+
+const CHUNK_SIZE = 255
+
+let isSigning = false
+
+let version: string
+let versionResolve
+
+const versionPromise = new Promise((resolve, reject) => {
+    versionResolve = resolve
+})
+
```

Comment:
Simple observable that keeps polling the ledger app, so that subscribers can be notified of status changes.